### DISCUSSION TITLE
The "path" argument must be of type string. Received type undefined

### DISCUSSION THREAD

**Discussion by kashifumar** [fwouts/previewjs#1069] [Q&A]
I just installed preview.js on Windows 10 Visual Studio Code and created a simple js file and then executed the extension but it gives me this error
The "path" argument must be of type string. Received type undefined
I am not able to find any reference for this anywhere

**Comment by fwouts** [comment]
Hi @kashifumar, please provide a sample repo to be able to reproduce 🙂

**Comment by fwouts** [comment]
Hey @kashifumar, this bug should be fixed in v1.15.1, please update Preview.js :)
